assembly/assembly-ide-war/pom.xml

```diff
@@ -48,7 +41,7 @@
         </dependency>
         <dependency>
             <groupId>org.eclipse.che</groupId>
-            <artifactId>assembly-ide-war</artifactId>
+            <artifactId>assembly-root-war</artifactId>
```

Comment:
maybe it makes sense to rename the folder also to `assembly-root-war` to make it consistent ?

Reply:
we could probably consider renaming as part of another PR

Reply:
I'm +1 on renaming, I didn't do it here to avoid messing with the build too much at once.

Reply:
agree PR was already BIG, we could handle it separately